GOT-StudentVersion/GOT-StudentVersion/Controllers/ViewController.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  ViewController.swift
+//  GOT-StudentVersion
+//
+//  Created by C4Q  on 11/2/17.
+//  Copyright © 2017 C4Q . All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController, UITableViewDelegate, UITableViewDataSource, UISearchBarDelegate {
+    
+    var Episodes = GOTEpisode.allEpisodes // constant
+    
+    @IBOutlet weak var tableView: UITableView!
+    
+    @IBOutlet weak var searchBarOutlet: UISearchBar!
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tableView.delegate = self
+        tableView.dataSource = self
+        searchBarOutlet.delegate = self
+        
+        
+    }
+    
+    
+    
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return 7
```

Comment:
the hard coding of this value is not good. 
You need to return as many episodes as you have if the amount of episodes changes and you keep this number your code will crash. 
 return episodes.count